packages/ui/src/components/ChannelNavSections.tsx

```diff
@@ -0,0 +1,89 @@
+import * as db from '@tloncorp/shared/dist/db';
+import { useCallback, useMemo } from 'react';
+
+import { SizableText, YGroup } from '../core';
+import ChannelListItem from './ChannelListItem';
+import ChannelNavSection from './ChannelNavSection';
+
+export default function ChannelNavSections({
+  group,
+  channels,
+  onSelect,
+  paddingBottom,
+}: {
+  group: db.GroupWithRelations;
+  channels: db.ChannelWithLastPost[];
+  onSelect: (channel: any) => void;
+  paddingBottom?: number;
+}) {
+  const hasSomeUnGroupedChannels = useMemo(
```

Comment:
Could this just be a check on the length of `ungroupedChannels`?

Reply:
Yep, fixed in latest commit.